media/css/mozorg/internet-health.scss

```diff
@@ -0,0 +1,830 @@
+$colorLiteracy: #00dc00; // green
+
+body {
+    border-top: none;
+    color: #56565A;
+}
+
+#outer-wrapper {
+    background: #fff;
+}
+
+main {
+    h1, h2, h3, h4, h5, h6 {
+        color: #56565a;
+        text-shadow: none;
+    }
+
+    p {
+        color: #56565a;
+    }
+}
+
+/*--------------------------------------------------------------------------*/
+// Page masthead
+
+#masthead {
+    background: #4a4a4a;
+    color: #fff;
+    padding: 0;
+    width: 100%;
+
+    .content {
+        position: relative;
+    }
+
+    .masthead-logo {
+        font-size: 100%;
+        height: 30px;
+        left: 0;
+        margin: 0;
+        padding: 10px 0 0;
+        position: absolute;
+        text-align: center;
+        top: 0;
+        width: 100%;
+
+        img {
+            height: 30px;
+        }
+    }
+
+    .download-button {
+        display: none;
+
+        .download-list {
+            margin-bottom: 0;
+        }
+
+        .download-link {
+            @include font-size(14px);
+            border-radius: 0;
+            border: none;
+            padding: 15px 20px;
+            text-transform: uppercase;
+        }
+    }
+
+    .fx-privacy-link {
+        display: none;
+    }
+
+    .masthead-nav-main {
+        .toggle {
+            background-color: darken(#4a4a4a, 10%);
+        }
+
+        .nav-main-menu a:hover {
+            border-color: $colorLink;
+            color: $colorLink;
+        }
+    }
+
+    @media #{$mq-tablet} {
+        .masthead-nav-main {
+            margin-right: 0;
+
+            .nav-main-menu {
+                padding: 15px 0;
+
+                a:link,
+                a:visited {
+                    color: #fff;
+                }
+
+                li {
+                    border-left: 1px solid #fff;
+
+                    &:first-child {
+                        border-left: none;
+                    }
+                }
+            }
+        }
+
+        .masthead-logo {
+            float: left;
+            padding: 10px 0;
+            position: relative;
+            width: auto;
+        }
+
+        .download-button {
+            display: block;
+            float: right;
+
+            .download-link {
+                padding: 15px 12px;
+            }
+        }
+    }
+
+    @media #{$mq-desktop} {
+        .download-button .download-link {
+            padding: 15px 20px;
+        }
+    }
+}
+
+html[dir="rtl"] {
+    @media #{$mq-tablet} {
+        #masthead .masthead-nav-main .nav-main-menu li {
+            border-left: none;
+            border-right: 1px solid #fff;
+
+            &:first-child {
+                border-right: none;
+            }
+        }
+    }
+}
+
+/*--------------------------------------------------------------------------*/
+// Main page header
+
+.main-heading-container {
+    @include background-size(cover, 100%);
+    background-color: $colorLink;
+    background-image: url('/media/img/mozorg/internet-health/header-small.jpg'),
```

Comment:
According to [caniuse](http://caniuse.com/#feat=multibackgrounds) IE9 doesn't support multiple backgrounds that contain both images and gradients. Is it worth making an image only fallback? (could we also just serve that to browsers than need the `-webkit` prefix, to avoid needing to repeat the multiple background rules twice?